Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -225,7 +226,13 @@ public function processRequest(ActionRequest $request, ActionResponse $response)
             $this->throwStatus(400, '400 Bad Request', 'Invalid HMAC submitted');
         }
 
-        $this->mapRequestArgumentsToControllerArguments();
+        try {
+            $this->mapRequestArgumentsToControllerArguments();
+        } catch (RequiredArgumentMissingException $e) {
+            $message = $this->throwableStorage->logThrowable($e);
+            $this->logger->notice('Request argument mapping failed due to a missing required argument. ' . $message, LogEnvironment::fromMethodName(__METHOD__));
+            $this->throwStatus(400, '400 Bad Request', 'Required argument is missing');
```

Comment:
The second argument `400 Bad Request` won't be used by `throwStatus()` because you specified `$content` (the third argument). Rather skip the third argument and use `Required argument is missing` (sic!) as the second parameter.

Reply:
Doh. Then why do the tests pass?

Reply:
Ah, I see. The 2nd argument is superflous the way I use it. I find that method behaviour very weird, I have to admit.

Reply:
I'd rather skip the second argument, because the content being "400 …" seems weird.